### PR TITLE
Remove daemon `blockr` since its service is no longer available

### PR DESCRIPTION
**Pull request by lemonlatte**
- Remove the `blockr` daemon

### CODE REVIEW

File: command/bitmark-wallet/coins.go [hunk 1]
```diff
@@ -141,8 +141,6 @@ func NewCoinCmd(coinType, short, long string, ct wallet.CoinType) *cobra.Command
 
 			var a agent.CoinAgent
 			switch agentData.Type {
```
Comment (nodestory) [inline command/bitmark-wallet/coins.go]: Remove the switch case? There is no other type.

**Comment by lemonlatte** [comment]
I tend to keep it because it is still an option in the command line. Or do you suggest to remove all related type flags and configurations from the CLI?

**Comment by nodestory** [comment]
If only `daemon` will be supported in the future, then I think we can even remove the option from the CLI.

**Comment by lemonlatte** [comment]
I do not know what will be supported in the future. It just design to multiple at beginning.

**Comment by nodestory** [comment]
It makes sense to have the switch clause because of CLI settings. It's fine to keep it in this way.

**Pull request merged**